installer/lib/alice_new/utilities.ex

```diff
@@ -0,0 +1,73 @@
+defmodule AliceNew.Utilities do
+  @moduledoc false
```

Comment:
Why no moduledoc?

Reply:
Because this is an internal module, not meant to be part of the public API.